src/components/Mailbox.vue

```diff
@@ -351,7 +369,9 @@ export default {
 				break
 			case 'del':
 				logger.debug('deleting', { env })
-				this.onDelete(env.databaseId)
+				if (e.srcKey === 'del' && this.hasDeleteAcl()) {
```

Comment:
this code is inside a switch case where e.shortKey matches `del`. I don't see a need for a second check. The first part of the if condition will always be `true`